specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,6318 @@
+      "description": "Hostpool update full properties including session host config, hostpool update parameters, and migration progress.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "updateStatus": {
+          "enum": [
+            "Validating",
+            "Scheduled",
+            "InProgress",
+            "Pausing",
+            "Paused",
+            "StartFailed",
+            "UpdateFailed",
+            "Cancelling",
+            "Canceled",
+            "Completed"
+          ],
+          "x-ms-enum": {
+            "name": "HostPoolUpdateStatus",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Validating",
+                "description": "Hostpool update validating."
+              },
+              {
+                "value": "Scheduled",
+                "description": "Hostpool update scheduled."
+              },
+              {
+                "value": "InProgress",
+                "description": "Hostpool update in progress."
+              },
+              {
+                "value": "Pausing",
+                "description": "Hostpool update pausing."
+              },
+              {
+                "value": "Paused",
+                "description": "Hostpool update paused."
+              },
+              {
+                "value": "StartFailed",
+                "description": "Hostpool update start failed."
+              },
+              {
+                "value": "UpdateFailed",
+                "description": "Hostpool update update failed."
+              },
+              {
+                "value": "Cancelling",
+                "description": "Hostpool update cancelling."
+              },
+              {
+                "value": "Canceled",
+                "description": "Hostpool update canceled."
+              },
+              {
+                "value": "Completed",
+                "description": "Hostpool update completed."
+              }
+            ]
+          },
+          "type": "string",
+          "description": "State of hostpool update.",
+          "x-nullable": false
+        },
+        "version": {
+          "type": "string",
+          "description": "The version of hostpool update."
+        },
+        "hostPoolResourceId": {
+          "type": "string",
+          "description": "The resourceId of hostpool."
+        },
+        "correlationId": {
+          "type": "string",
+          "description": "The correlationId of the hostpool update."
+        },
+        "migrateProgress": {
+          "$ref": "#/definitions/HostPoolUpdateProgress",
+          "description": "Properties of host pool update progress."
+        },
+        "hostPoolUpdateParameters": {
+          "$ref": "#/definitions/HostPoolUpdateParameters",
+          "description": "Represents properties for a hostpool update."
+        },
+        "sessionHostConfigurations": {
```

Comment:
Why plural?  It is one configuration, correct?

Reply:
will change to sessionHostConfiguration